site/content/docs/concepts/pipelines.en.md

```diff
@@ -12,19 +12,19 @@ We won't get too philosophical about releasing software, but what's the point of
 
 Using an automated release tool like CodePipeline helps make your release manageable. Even if your release isn't particularly complicated, knowing that you can just `git push` to deploy your change always feels a little magical 🌈.
 
-## Pipeline structure
+## Pipeline Structure
 
 Copilot can set up a CodePipeline for you with a few commands - but before we jump into that, let's talk a little bit about the structure of the pipeline we'll be generating. Our pipeline will have the following basic structure:
 
 1. __Source Stage__ - when you push to a configured GitHub, Bitbucket, or CodeCommit repository branch, a new pipeline execution is triggered.
-2. __Build Stage__ - after your source code is pulled from your repository host, your service's container image is built and published to every environment's ECR repository.
+2. __Build Stage__ - after your source code is pulled from your repository host, your service's container image is built and published to every environment's ECR repository and any environment files are uploaded to S3.
```

Comment:
Does `"environment files"` here refers to addon templates and the [environment variable files](https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#env_file)? What do you think about saying something like `"...and any artifacts, such as your addon templates and your environment variable files..."`